sdks/python/tests/library_integration/langchain/test_langchain.py

```diff
@@ -750,3 +751,27 @@ def f():
     assert len(fake_backend.span_trees) == 1
     assert len(callback.created_traces()) == 0
     assert_equal(EXPECTED_SPANS_TREE, fake_backend.span_trees[0])
+
+
+def test_langchain_callback__disabled_tracking(fake_backend):
+    os.environ["OPIK_TRACK_DISABLE"] = "true"
```

Comment:
You should use `testlib.patch_environ` context manager for that to make sure the environment will return back to its initial state after the test has finished (no matter how).

Otherwise, if this test fails in the middle, it may affect the environment and can potentially hard other tests (especially with THAT environment variable :).